src/index.ts

```diff
@@ -213,6 +213,9 @@ export const round = (x1: string | number, precision: number): string =>
 
 export function toBns(n: number | string): string {
   let out = typeof n === 'number' ? n.toString() : n.replace(/^\s+|\s+$/g, '')
+  if (out === '') {
+    out = '0'
+  }
```

Comment:
This should be an error case rather than coercing white-space character strings to a big number string

Reply:
biggystring already handles empty string as zero and there are tests for it, for better or worse.

Reply:
I guess this has been to spec for a while. 🤷‍♂️